### DISCUSSION TITLE
how to compile orafce

### DISCUSSION THREAD

**Discussion by ghost** [termux/termux-packages#18980] [Q&A] [answered]
https://github.com/orafce/orafce
This is used in PostgreSQL databases to be more compatible with code migrated from Oracle.
I tried to compile it

curl -LJO https://github.com/orafce/orafce/archive/refs/tags/VERSION_4_9_0.tar.gz
tar -xzvf orafce-VERSION_4_9_0.tar.gz
cd orafce-VERSION_4_9_0/
make
make install

the install fails with

/usr/bin/mkdir -p '/data/data/com.termux/files/usr/lib/postgresql'
make: /usr/bin/mkdir: No such file or directory
make: *** [/data/data/com.termux/files/usr/lib/postgresql/pgxs/src/makefiles/../../src/Makefile.shlib:430: installdirs-lib] Error 127

How can I fix this?

**Comment by sylirre** [comment, marked as answer]
Well, calling executable `/usr/bin/mkdir` not going to work. Either edit paths or use `termux-chroot` utility (package `proot`). Explained at https://wiki.termux.com/wiki/Differences_from_Linux
